inst/ms/aggreCAT.qmd

```diff
@@ -775,8 +844,7 @@ argument. Aggregation methods requiring supplementary data, include
 `QuizWAgg` applied with **TODO: what wrapper function??** and
```

Comment:
QuizWAgg is done with LinearWAgg and the participant type for user supplied weights. Doesn't exist as it's own method type anymore since there is nothing to distinguish it from any other linear weight

Reply:
Yup, thanks! Should I bother mentioning this in the text / method summary table at all?

Reply:
Same goes for mentioning EngWAgg:

https://github.com/metamelb-repliCATS/aggreCAT/blob/ba49150704a4d28f8b0f38112909c9131bbcc166/inst/ms/aggreCAT.qmd#L833-L835

Reply:
Nah, I think you can skip any specific method using user-defined weights. Or at most a sentence saying that things like QuizWAgg from the Hanea paper are fit in aggreCAT using the user-supplied weights method.